src/scripts/heroAnimation/hero.js

```diff
@@ -0,0 +1,256 @@
+let width,
+  height,
+  ctx,
+  points,
+  target,
+  canvasTop,
+  animateHeader = true,
+  timer = null;
+const canvas = document.getElementById("js-canvas-hero");
+const container = document.querySelector(".grav-c-hero");
+
+function HeroAnimation() {
+  if (container !== null) {
+    addListeners();
+    initHeader();
+    initAnimation();
+  }
+}
+
+function resetTimer() {
+  window.clearTimeout(timer);
+  timer = null;
+  timer = window.setTimeout(initHeader, 60000);
```

Comment:
What about writing these interval in a syntax like `60 * 1000`? I used to write the actual number too, but it makes sense to use the `*` operator.

Reply:
Hm my maths isn't exactly the best and I didn't feel it's necessary, milliseconds are pretty straightforward IMHO